test/e2e/antreapolicy_test.go

```diff
@@ -1861,6 +1888,19 @@ func applyClusterGroups(t *testing.T, step *TestStep) {
 	}
 }
 
+func cleanupTestStepClusterGroups(t *testing.T, step *TestStep) {
+	for _, g := range step.Groups {
+		if cg, ok := g.(*corev1a2.ClusterGroup); ok {
+			log.Debugf("deleting CG %v", cg.Name)
+			failOnError(k8sUtils.DeleteCG(cg.Name), t)
+		}
+	}
+	if len(step.Groups) > 0 {
+		log.Debugf("Sleeping for %v for all group deletions to take effect", networkPolicyDelay)
+		time.Sleep(networkPolicyDelay)
```

Comment:
i feel like group deletions should be fast enough since they are not really pushed to the agents and data path.. you think we can get rid of the delay?

Reply:
Done. Also added a new smaller groupDelay in place of the networkPolicyDelay, as I think group membership computation would be much faster than policy enforcement.

Reply:
Done. Also added a new smaller `groupDelay` in place of the `networkPolicyDelay`. I think the group membership computation should be much faster than policy enforcement.